lib/hammer_cli_foreman_tasks/helper.rb

```diff
@@ -3,10 +3,14 @@ module Helper
 
     # render the progress of the task using polling to the task API
     def task_progress(task_or_id)
-      task_id = task_or_id.is_a?(Hash) ? task_or_id['id'] : task_or_id
-      task_progress = TaskProgress.new(task_id) { |id| load_task(id) }
-      task_progress.render
-      task_progress.success?
+      if !options.values.delete_if(&:empty?).empty?
+        task_id = task_or_id.is_a?(Hash) ? task_or_id['id'] : task_or_id
+        task_progress = TaskProgress.new(task_id) { |id| load_task(id) }
+        task_progress.render
+        task_progress.success?
+      else
+        STDERR.puts "Error: Please mention appropriate attribute value"
```

Comment:
@iNecas @amitkarsale if you can't test the value in `validate_options` than you can use `signal_usage_error(_("Please mention appropriate attribute value"))`. It's a helper method from Clamp that raises `UsageError`. Hammer catches it and prints:
```
Error: Please mention appropriate attribute value

See: 'hammer host list --help'
```

Also please note that we translate errors that can bubble up to users.